harvester_e2e_tests/scenarios/test_vm_negatives.py

```diff
@@ -65,6 +66,24 @@ def test_host_shutdown(self, request, admin_session,
     @pytest.mark.host_management
     def test_delete_vm_while_host_shutdown(self, request, admin_session,
                                            harvester_api_endpoints, basic_vm):
+        def _check_vm_instance_nodename():
+            resp = admin_session.get(
+                harvester_api_endpoints.get_vm_instance % (
```

Comment:
suggest using utils.lookup_vm_instance(). Also make `vm_instance_josn` and `nonlocal` inside the `_check_vm_instance_nodename()` function so we don't need line 88-89 anymore.